packages/resolver/src/translators/reactjs/hooks/useResolverCryptoAddresses.tsx

```diff
@@ -23,13 +21,13 @@ export const useResolverCryptoAddresses = (
   const {crypto} = useResolver()
 
   const query = useQuery({
-    throwOnError: true,
     queryKey: ['useResolverCryptoAddresses', resolve],
+    queryFn: async ({signal}: {signal?: AbortSignal}) => {
```

Comment:
signal in this case is mandatory, cuz when the debounce of typing kicking again, it should cancel the previous request
```suggestion
    queryFn: async ({signal}: {signal: AbortSignal}) => {
```